src/w_wad.c

```diff
@@ -240,10 +240,9 @@ static void W_CoalesceMarkedResource(const char *start_marker,
 
   if (mark_end)                                   // add end marker
     {
-      size_t len = strnlen(end_marker, 8);
       lumpinfo[numlumps].size = 0;  // killough 3/20/98: force size to be 0
       lumpinfo[numlumps].namespace = ns_global;   // killough 4/17/98
-      memcpy(lumpinfo[numlumps++].name, end_marker, len < 8 ? len + 1 : len);
+      M_CopyLumpName(lumpinfo[numlumps++].name, end_marker);
```

Comment:
How about line 514?

Reply:
Done in https://github.com/fabiangreffrath/woof/pull/1012/commits/49b11e92c4a8f4c15fb3ad36358896c39de272ac We probably have a lot of lump names in other places.